db/migrate/20210701000400_add_user_id_to_participant_declarations.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class AddUserIdToParticipantDeclarations < ActiveRecord::Migration[6.1]
+  disable_ddl_transaction!
+
+  def change
+    add_reference :participant_declarations, :user, null: true, type: :uuid, index: { algorithm: :concurrently }
```

Comment:
Why `:concurrently`?

Reply:
Again. A stong_migrations thing.

Specifically, when you run it without both those lines it give:

```
=== Dangerous operation detected #strong_migrations ===

Adding an index non-concurrently locks the table. Instead, use:

class AddUserIdToParticipantDeclarations < ActiveRecord::Migration[6.1]
  disable_ddl_transaction!

  def change
    add_reference :participant_declarations, :user, null: true, type: :uuid, index: {algorithm: :concurrently}
  end
end

```

Reply:
Fair enough